client/src/components/ApplicantProfile.js

```diff
@@ -23,23 +23,38 @@ const ApplicantProfile = () => {
   ]);
 
   const [profileCreated, setProfileCreated] = useState(null);
+  const addMoreExperience = () => {
+    setExperienceData([
+      ...experienceData,
+      {
+        company: "",
```

Comment:
this could be extracted into a const defaultExperience = {
  company: "",
  ...
}

and then used at lines 16 and 26 as {...defaultExperience} (to create a new copy each time).